posts/templates/posts/partials/_stream_post.html

```diff
@@ -12,6 +14,13 @@ <h3>{{ post.title }}</h3>
             <p>
                 {{ post.content }}
             </p>
+        {% elif post.content_type == 'text/markdown' %}
+            <!-- TODO: Decide how to display markdown on the stream -->
```

Comment:
How about we just display the description on the stream to make it "streamlined", also there's little to no difference between viewing a post in stream vs on the detailed page hahahah

Reply:
Do we want to remove this in a follow up PR? I don't mind